substrate/frame/revive/uapi/src/host.rs

```diff
@@ -518,24 +504,6 @@ pub trait HostFn: private::Sealed {
 	/// - [Sr25519VerifyFailed][`crate::ReturnErrorCode::Sr25519VerifyFailed]
 	#[unstable_hostfn]
 	fn sr25519_verify(signature: &[u8; 64], message: &[u8], pub_key: &[u8; 32]) -> Result;
-
```

Comment:
not sure why this is re-ordered here, this make the diff bigger than what it could be